prisma/schema.prisma

```diff
@@ -0,0 +1,17 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model users {
+  id Int @id @default(autoincrement())
+  login String @unique
+  email String @unique
+}
```

Comment:
Please create another model, it can be anything(posts, events, cars, robots, events) but it should be related to users